pkg/clusters/addons/kong/kong.go

```diff
@@ -296,3 +336,29 @@ func urlForService(ctx context.Context, cluster clusters.Cluster, nsn types.Name
 
 	return nil, fmt.Errorf("service %s has not yet been provisoned", service.Name)
 }
+
+func deployKongEnterpriseLicenseSecret(ctx context.Context, cluster clusters.Cluster, namespace, name string) error {
+	license := os.Getenv(EnterpriseLicense)
+	if license == "" {
+		return fmt.Errorf("failed retrieving license key from environment")
+	}
```

Comment:
Related to comments above, we should make this a function argument rather than an env var:

```suggestion
func deployKongEnterpriseLicenseSecret(ctx context.Context, cluster clusters.Cluster, namespace, name, license string) error {
	if license == "" {
		return fmt.Errorf("failed retrieving license key from environment")
	}
```

Reply:
e02d137c22f069e1af34af42d9e3367a1e7e20b3